### PR TITLE
US04_Vehicle_Contracts_Adri corrected

### PR DESCRIPTION
**Pull request by arosta16**

### CODE REVIEW

File: src/test/java/com/team_elite_sprint2/pages/Vehicle_Contracts_page.java [hunk 1]
```diff
@@ -5,9 +5,9 @@
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.PageFactory;
 
-public class US04_Vehicle_Contracts_page extends BasePage{
+public class Vehicle_Contracts_page extends BasePage{
```
Comment (Karshyga1985) [inline src/test/java/com/team_elite_sprint2/pages/Vehicle_Contracts_page.java:8]: Adri, I think that would be great to remove "_" in the page name, then we will use one naming convention for pages as a team. Thank you!

File: src/test/resources/features/US04_Vehicle_Contracts.feature [hunk 2]
```diff
@@ -4,19 +4,19 @@ Feature: Default
...
+			|store manager|
 
 	
 	@B32G14-200
-	Scenario: Verify that Drivers can NOT access the Vehicle Contracts page 
+	Scenario: Verify that Drivers can NOT access the Vehicle Contracts page
 		Given the user logged in as "driver"
-		When user chooses FLEET menu option on the Fleet Management page
-		And user clicks on Vehicle Contracts from dropdown menu
+		When user hover over on Fleet and clicks Vehicle Contracts
```
Comment (Karshyga1985) [inline src/test/resources/features/US04_Vehicle_Contracts.feature:21]: the same suggestion for this line as above. You did a great job!

**Review by Karshyga1985 — APPROVED** [review]
Looks good! Great job!

File: src/test/java/com/team_elite_sprint2/pages/Vehicle_Contracts_page.java (hunk 1 above)
Comment (ChristinesSok) [reply src/test/java/com/team_elite_sprint2/pages/Vehicle_Contracts_page.java]: I agree with Kara on the naming convention, I do like it to be the same as the team. New name page would be "VehicleContractsPage" other than that I think you are looking good.

**Review by ChristinesSok — APPROVED** [review]
Looks good, just the page name could be in line with team. Other than that I think great job.

**Pull request merged**